NEWS.md

```diff
@@ -1,11 +1,12 @@
 # Release Notes
 
-## Version 3.2.4 (Release Candidate 1)
+## Version 3.2.4 (August 6, 2022)
 
 ### Corrections
 
  * Fixed a regression where shared folders could not be added using the Preferences dialog
  * Fixed a performance regression when updating file transfer lists
```

Comment:
... `on some systems`

Reply:
As far as I can tell, it affects all systems, including mine, when multiple transfers are active.